src/Compilers/CSharp/Test/Symbol/Symbols/Source/RecordTests.cs

```diff
@@ -161,7 +161,7 @@ public void GeneratedProperties()
             Assert.Equal(c, x.ContainingType);
             Assert.Equal(c, x.ContainingSymbol);
 
-            var backing = x.BackingField;
+            var backing = x.BackingField!;
```

Comment:
>! [](http://example.com/codeflow?start=40&length=1)

If I were you, I would disable nullable analysis for the entire file instead. It really doesn't help much in tests, quite the opposite. It forces us to do some extra work to suppress warnings one way or another. The test is going to fail anyway if something is wrong. It is not meant to handle every possible scenario, just specific scenario we have at hand.